packages/vaadin-combo-box/test/keyboard.test.js

```diff
@@ -232,74 +225,74 @@ describe('keyboard', () => {
     it('should close the overlay with escape if there is no focus', () => {
       comboBox._focusedIndex = -1;
 
-      esc();
+      escKeyDown(comboBox.inputElement);
 
       expect(comboBox.opened).to.equal(false);
     });
 
     it('escape key event should not be propagated', () => {
-      const listener = document.body.addEventListener('keydown', (e) => {
-        if (e.keyCode == 27) {
-          throw new Error('Escape key was propagated to body');
-        }
-      });
+      const spy = sinon.spy();
 
-      esc();
+      const listener = document.body.addEventListener('keydown', spy);
```

Comment:
Why do we need `listener` here? This could be simplified as follows:

```js
document.body.addEventListener('keydown', spy);
```

And then below we can remove listener:

```js
document.body.removeEventListener('keydown', spy);
```

Reply:
Good point! Fixed.